operator_ui/src/components/Bridges/Form.tsx

```diff
@@ -1,42 +1,63 @@
-import React from 'react'
-import PropTypes from 'prop-types'
-import { Prompt } from 'react-router-dom'
-import * as formik from 'formik'
-import { withStyles } from '@material-ui/core/styles'
-import { TextField, Grid } from '@material-ui/core'
+import { Grid, TextField } from '@material-ui/core'
+import {
+  createStyles,
+  Theme,
+  withStyles,
+  WithStyles
+} from '@material-ui/core/styles'
 import Button from 'components/Button'
-import { set, get } from 'utils/storage'
+import formik, { FormikProps } from 'formik'
```

Comment:
`Attempted import error: 'formik' does not contain a default export (imported as 'formik').`
I was about to suggest importing `{withFormik, Form as FormikFormOrSomeOtherName}` without this build error too.

Reply:
Hm, lets keep it as a default import for now, since making use of the namespace will solve the naming ambiguity anyway. About to push up a PR that fixes the build error. Will need to double check with you and @rupurt that my fix is correct since you guys have more context on the babel ecosystem than I do at this point in time.

Reply:
70cbefc